CONTRIBUTING.md

```diff
@@ -28,6 +28,10 @@ issues first, as it's possible that someone else reported the same issue before
 you. Though it helps save time, don't worry! We won't mind if you accidentally
 post a duplicate issue.
 
+Please note that if an issue can only be reproduced on a nightly compiler it will be closed and marked "Will not fix."
```

Comment:
I agree with @kvark and @torkleyy, it's too unfriendly to new users. We'll miss a lot of bugs in this case.